### ISSUE TITLE
Type of `**kwargs` that uses an unpacked `TypedDict` is revealed incorrectly

### ISSUE THREAD

**Issue by erictraut** [microsoft/pyright#9199]
The following code should not generate an error:

```python
from typing import TypedDict, Unpack, assert_type

class TD1(TypedDict):
    x: int

def func2(v3: str, **kwargs: Unpack[TD1]) -> None:
    assert_type(kwargs, TD1)
```

This is due to a recent regression. It doesn't affect any common use cases, but it does cause a false positive in the Python typing spec conformance tests, so it should be fixed.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.385.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.386
